invenio_administration/assets/semantic-ui/js/invenio_administration/src/components/utils.js

```diff
@@ -21,3 +20,22 @@ export const sortFields = (schema) => {
       }, {})
   );
 };
+
+export const deserializeFieldErrors = (errors) => {
```

Comment:
Validation errors from the backend have the following format:

```javascript
errors: [
  {
    "field": "foo",
    "message": "Foo must be greater than Bar"
  },
  {
    "field": "doe",
    "message": "Doe must be greater than Bar"
  }
]
```

Therefore, this must be parsed and passed to Formik's bag [setErrors](https://formik.org/docs/api/formik#seterrors-fields--field-string-string---void) method.

This approach is based on `react-invenio-deposit`:

https://github.com/inveniosoftware/react-invenio-deposit/blob/master/src/lib/DepositBootstrap.js#L103
https://github.com/inveniosoftware/react-invenio-deposit/blob/master/src/lib/DepositRecordSerializer.js#L349-L362